File: src/main/java/com/example/pescAstur/model/Product.java
package com.example.pescAstur.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class Product {
    private int cantidadStock;
    private String categoria;
    private double coste;
    private String descripcion;
    private double descuento;
    private String dimensiones;
    private String fechaCreacion;
    private String imagenURL;
    private String marca;
    private String nombre;
    private String peso;
    private double precio;
    private List<Integer> rating;
    private List<String>comentarios;
    private String UID;
}

File: src/app/model/product.ts
export class Product {
    cantidadStock:number;
    categoria:string;
    coste:number;
    descripcion:string;
    descuento:number;
    dimensiones:string;
    fechaCreacion:string;
    imagenURL:string;
    marca:string;
    modelo:string;
    nombre:string;
    peso:string;
    precio:number;
    rating:number[];
    comentarios: string[];

    constructor(){
        this.cantidadStock = 0;
        this.categoria = '';
        this.coste = 0;
        this.descripcion = '';
        this.descuento = 0;
        this.dimensiones = '';
        this.fechaCreacion = '';
        this.imagenURL = '';
        this.marca = '';
        this.modelo = '';
        this.nombre = '';
        this.peso = '';
        this.precio = 0;
        this.rating =[];
        this.comentarios = [];
    }
}
